nectar/config.py

```diff
@@ -124,6 +125,7 @@ def __init__(
         self._temp_files = []
         self.connect_timeout = connect_timeout
         self.read_timeout = read_timeout
+        self.working_dir = working_dir
```

Comment:
Consider:
```
DownloadConfig(object):
    ...
    WORKING_DIR = '/tmp'
    ...

def __init__(self, ..., working_dir=None):
    ...
    self.working_dir = working_dir or DownloadConfig.WORKING_DIR
   ...
```
Thinking this would support changing the global default to the working directory in the platform code before calling into the plugin.  This way we don't need to update everywhere in plugin code that creates a DownloadConfig and have it pass the working directory.  Just a thought.